src/events/user.js

```diff
@@ -212,7 +212,7 @@ function userEvents(postmark) {
 
     calEvent
       .create(session)
-      .then(result => dlog('Event created %d, %O', result.status, result.data))
+      .then(result => dlog('Event created %d, %o', result.status, result.data))
```

Comment:
Just don't care about the return from the promise?

Reply:
That's right, we do nothing with it. It is nice for verification when testing locally.